src/app/organizations/organization-details/manage-projects/manage-single-project/manage-single-project.component.html

```diff
@@ -9,7 +9,7 @@
                 <div class="app-page-header flex__center">
                     <div class="app-page-title-back" (click)="backToOrganizationDetailsScreen()">
                         <i class="fas fa-long-arrow-alt-left mr-3"></i>
-                        {{ states.project.organization.name }}
+                        {{ states.project.organization?.name }}
```

Comment:
This is just a temporary fix because the backend doesn't return the organization when a project gets updated.